### PR TITLE
Wse 67

### PR DESCRIPTION
**Pull request by rgr182**

### CODE REVIEW

File: ScraperAdmin/Media/Git.png
Comment (Gapcroz) [inline ScraperAdmin/Media/Git.png:1]: it's ok to added this .png??

**Review by Gapcroz — COMMENTED** [review]
it's necesary to add .png file?

File: ScraperAdmin/Media/Git.png
Comment (rgr182) [reply ScraperAdmin/Media/Git.png:1]: is just a base demo image later we-ll adapt it

File: ScraperAdmin/DataAccess/Models/Scraper.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace ScraperAdmin.DataAccess.Models
+{
+    public class Scraper
+    {
+        public Guid ScraperId { get; set; }
+        public ScraperStatus ScraperStatusId { get; set; }
+        public string ImagePath { get; set; }
+        public DateTime? LastExecutionDate { get; set; }
+        public string ScrapperName { get; set; }
```
Comment (Paroxyse) [inline ScraperAdmin/DataAccess/Models/Scraper.cs]: typo

**Review by Paroxyse — APPROVED** [review]

**Pull request merged**